src/GraphQL/GraphQLExtensions.cs

```diff
@@ -151,7 +153,7 @@ internal static (IGraphType resolvedType, Type type) GetNamedTypes(this IGraphTy
         public static IGraphType GetNamedType(this IGraphType type)
         {
             if (type == null)
-                return null;
+                return null!;
```

Comment:
:)

Reply:
For backwards compatibility I left it like this.  For v5 we could make this method throw an exception if `type` is null.  Or we could add nullability attributes that specify that when `type` is null, the return value will be also be null.